packages/matplotlib/patches/fix-freetype.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/setupext.py b/setupext.py
+--- a/setupext.py
++++ b/setupext.py
+@@ -522,6 +522,9 @@ class FreeType(SetupPackage):
+     name = "freetype"
+ 
+     def add_flags(self, ext):
++        if "PYODIDE_PACKAGE_ABI" in os.environ:
++            ext.libraries.append('freetype')
++            return
```

Comment:
Could you contribute this patch upstream?

Reply:
Sure.